package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "@jitsi/web-sdk",
+    "version": "0.0.0",
+    "description": "Web SDK for the Jitsi Meet IFrame",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "scripts": {
+        "build": "tsc",
+        "lint": "eslint --ext .ts,.tsx,.js ./src",
+        "lint-fix": "npm run lint -- --fix",
+        "test": "jest --config jestconfig.json"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/jitsi/jitsi-meet-web-sdk.git"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "Apache-2.0",
```

Comment:
Please add a LICENSE file with Apache 2 too please.

Reply:
Isn't [this](https://github.com/jitsi/jitsi-meet-web-sdk/blob/main/LICENSE) enough?

Reply:
LoL I forgot I added that!